src/derivatives/atlas.md

```diff
@@ -155,22 +151,22 @@ A guide for using macros can be found at
    "mni152nlin2009casym-pipeline": {
       "tpl-MNI152NLin2009cAsym": {
          "anat": {
-            "tpl-MNI152NLin2009cAsym_res-1_label-brain_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-eye_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-face_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-head_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-CSF_probseg.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-GM_probseg.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-WM_probseg.nii.gz": "",
+            "tpl-MNI152NLin2009cAsym_res-1_seg-brain_mask.nii.gz": "",
+            "tpl-MNI152NLin2009cAsym_res-1_seg-eye_mask.nii.gz": "",
+            "tpl-MNI152NLin2009cAsym_res-1_seg-face_mask.nii.gz": "",
+            "tpl-MNI152NLin2009cAsym_res-1_seg-head_mask.nii.gz": "",
+            "atlas-MNI152NLin2009cAsym_res-1_seg-CSF_probseg.nii.gz": "",
+            "atlas-MNI152NLin2009cAsym_res-1_seg-GM_probseg.nii.gz": "",
+            "atlas-MNI152NLin2009cAsym_res-1_seg-WM_probseg.nii.gz": "",
```

Comment:
Okay, I think I understand how you would use atlas- at the front. Just because this is not a mask, but encoding more regions at once now it becomes an atlas. I think you don't want that. These are just segmentations of a template. I believe, a segmentation becomes an atlas when there's a rationale behind that partition of the space, typically because of some regional property of the brain.

tpl here is still giving the same information as above for masks. The change of data type is already given by the suffix. Adding here atlas is very confusing, especially because it now conflates the space designation id.
```suggestion
            "tpl-MNI152NLin2009cAsym_res-1_label-CSF_probseg.nii.gz": "",
            "tpl-MNI152NLin2009cAsym_res-1_label-GM_probseg.nii.gz": "",
            "tpl-MNI152NLin2009cAsym_res-1_label-WM_probseg.nii.gz": "",
```